tools/query_verification/src/main/java/com/google/bigquery/BigQueryManager.java

```diff
@@ -279,8 +281,55 @@ public String getName() {
             StandardSQLTypeName type = fields.get(i).getType().getStandardType();
 
             Object result;
-            // TODO Convert value to appropriate Java type
-            result = value.getStringValue();
+            try {
+                switch (type) {
```

Comment:
Do we need to have a default to at least put some warnings for unsupported types.

Reply:
I have a default case that converts the value to a string to use for the other types not listed. Should I still add a warning?

Reply:
Adding a warning sounds good, since it can help us identify the unsupported types.